src/components/user/userAccountStorage.ts

```diff
@@ -0,0 +1,14 @@
+import * as vscode from 'vscode';
+
+export class GlobalStateAccountStorage {
+	constructor(private readonly globalState: vscode.Memento) {}
```

Comment:
I usually write it like `__globalState`

Reply:
so, for all private vars we should use `__` prefix, right?